patch/kernel/archive/rockchip64-6.19/rk3308-acodec-vendor-driver.patch.disabled.needs.entirely.rebased

```diff
@@ -0,0 +1,7156 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```

Comment:
does not align anymore due to upstream renaming of functions as it seems. horrible task to align manually again...